lib/cocoapods/target/aggregate_target.rb

```diff
@@ -290,7 +290,7 @@ def podfile_dir_relative_path
     # @return [String] The path of the xcconfig file relative to the root of
     #         the user project.
     #
-    def xcconfig_relative_path(config_name)
+    def xcconfig_relative_srcroot(config_name)
```

Comment:
why was this method renamed?

Reply:
I remove the `xcconfig_relative_path` and add a `xcconfig_relative_pods_root`. But I found that the xcconfig should get the path which relative user project. So I revert the changes. But I want to make a different with `Pod Root`, I change the name to `xcconfig_relative_srcroot`.

Reply:
I think keeping the name and just updating the function doc would be OK - specifically the section on what the function returns